cli/src/commands/start.rs

```diff
@@ -128,9 +128,13 @@ pub struct Start {
     /// Specify the path to the file where logs will be stored
     #[clap(default_value_os_t = std::env::temp_dir().join("snarkos.log"), long = "logfile")]
     pub logfile: PathBuf,
+
     /// Enables the metrics exporter
     #[clap(default_value = "false", long = "metrics")]
     pub metrics: bool,
+    /// Specify the IP address and port for the metrics exporter
+    #[clap(long = "metrics-listen")]
```

Comment:
Maybe `metrics-ip` would be a more appropriate name.

```suggestion
    #[clap(long = "metrics-ip")]
```